Vizzuality/forest-atlas-landscape-cms
# Update the styles of the widgets when inserted in Open Content pages

This PR updates the visual styles of the widgets inserted in Open Content pages and enable the users to toggle on and off their border.

<p align="center">
<img width="381" alt="Widget with a border around it and updated styles" src="https://user-images.githubusercontent.com/6073968/72824098-9f999b00-3c6c-11ea-99bb-e67c55fb40e1.png">
</p>

Widgets that were already inserted in Open Content pages (prior to this PR) won't display any border, though the user can decide to toggle it on. Widgets inserted from now on will have a border by default (which the user can toggle off).

If the widgets have a description, it is displayed between the title and the visualisation. If they have a citation, it is displayed below it. If the user has allowed their download, a button will also be displayed below the visualisation.

## Testing instructions

1. Go to any Open Content page that contains a widget

Make sure the widget doesn't have a border around it.

2. Go and edit the page
3. Add a new widget

Make sure the new widget has a border around it.

4. Toggle off the border by clicking the checkbox in the “Configuration” box, below the widget

Make sure the new widget doesn't have a border any more.

5. Edit one of the widgets you inserted in the page
6. Add a description, citation and allow their download (depends on #423)
7. Go again to the Open Content page

Make sure the description, citation and download button are visible.

## Pivotal Tracker

[Task](https://www.pivotaltracker.com/story/show/170233515).
